config/services.yaml

```diff
@@ -24,4 +24,28 @@ services:
         tags: ['controller.service_arguments']
 
     # add more service definitions when explicit configuration is needed
-    # please note that last definitions always *replace* previous ones
\ No newline at end of file
+    # please note that last definitions always *replace* previous ones
+
+    App\Controller\CacheController:
+        autowire: false
+
+    App\EventListener\HttpCacheListener:
```

Comment:
By implementing the `EventSubscriberInterface` you should be able to get rid of the `tags` definition

Reply:
Well, now I'm getting this error:
```
The service "App\EventListener\HttpCacheListener" (class: App\EventListener\HttpCacheListener) must return an event for each subscribed event.
```

Reply:
Ok it works fine now